app/src/forms/submission/routes.js

```diff
@@ -57,6 +57,14 @@ routes.post('/:formSubmissionId/email', hasSubmissionPermissions([P.SUBMISSION_R
   await controller.email(req, res, next);
 });
 
+routes.get('/:formSubmissionId/emailRecipients', hasSubmissionPermissions([P.SUBMISSION_REVIEW]), async (req, res, next) => {
+  await controller.getEmailRecipients(req, res, next);
+});
+
+routes.post('/:formSubmissionId/emailRecipients', hasSubmissionPermissions([P.SUBMISSION_REVIEW]), async (req, res, next) => {
+  await controller.addEmailRecipients(req, res, next);
+});
+
```

Comment:
It would be great to have new routes added to `app/src/docs/v1.api-spec.yaml` - but completely understood if you don't have time! We need to clean up the docs at some point and it can be done then.

Reply:
Oops. I had intended to update the docs, but it completely slipped my mind. I've added the new routes now.